src/test_bubble.py

```diff
@@ -0,0 +1,41 @@
+"""Test bubble sort."""
+
+
+import pytest
+from random import randint
+
+
+to_sort = [
+    ([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]),
+    ([10, 9, 8, 7, 6, 5, 4, 3, 2, 1], [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]),
+    ([10, 9, 8, 17, 6, 300, 4, 3, 2, 11], [2, 3, 4, 6, 8, 9, 10, 11, 17, 300])
+]
```

Comment:
What about when there are duplicates?
What about when all numbers are the same?
What if you only have 2 numbers?
What if you have 1000 numbers?

Does it still work in all these circumstances?